src/test/java/org/helpberkeley/memberdata/RoutedDeliveriesTest.java

```diff
@@ -40,10 +44,25 @@ public RoutedDeliveriesTest() {
         users = new Tables(loader.load()).mapByUserName();
     }
 
+    @AfterClass
+    public static void cleanup() throws IOException {
+        cleanupGeneratedFiles();
+    }
```

Comment:
You may also want a @Before annotation that calls this, to run it in between every test method.
The @AfterClass is just for cleanup after the last test has been run.